src/TreeDumpLibrary/VisualTreeDumper.cs

```diff
@@ -333,6 +335,32 @@ public string PropertyValueToString(string propertyName, object propertyObject)
                 Size size = (Size)propertyObject;
                 return $"[{(int)size.Width}, {(int)size.Height}]";
             }
+            else if(propertyObject is IList<TextHighlighter>)
+            {
+                var sb = new StringBuilder();
+                sb.Append("[");
+
+                foreach (var th in propertyObject as IList<TextHighlighter>)
+                {
+                    var background = (SolidColorBrush)th.Background;
+                    sb.Append("\"(" + background.Color.ToString() + ",");
+
```

Comment:
I think this should just call PropertyValueToString recursively on th.
(This would enable it to work on non-solidcolor brushes, e.g. if the background is set to a gradient brush)

Reply:
If I call it recursively, this if-branch will run:
```
else if (propertyObject is SolidColorBrush)
{
    Quote((propertyObject as SolidColorBrush).Color.ToString());
}
```
so I still won't support gradients, right?